concordium-std/src/types.rs

```diff
@@ -368,7 +375,7 @@ macro_rules! claim_ne {
 ///
 /// # Example
 /// Defining a custom error type
-/// ```rust
+/// ```ignore
```

Comment:
This should generally not be done. It makes docs less nice to look at (e.g., worse syntax highlighting). 

THe correct way to fix this is to make it compile, but that should not be part of this PR.

Please revert this.